README.md

```diff
@@ -27,7 +27,13 @@
 
 - Visit [SQL Try Editor at W3Schools.com](https://www.w3schools.com/Sql/tryit.asp?filename=trysql_select_top) and write the following queries:
   - find all customers with a particular first name.
+  * SELECT
   - find all customers that live in London.
+
+  * select country, *
```

Comment:
Oddly enough, no customers live in London.  I think they may have meant UK.  But good job.